### DISCUSSION TITLE
5.0.0 DryIoc.nupkg

### DISCUSSION THREAD

**Discussion by hymccord** [dadhi/DryIoc#469] [General]
With the 5.0.0 release, is the support for the nuget package named "DryIoc" (that uses content files instead of the compiled dll) being deprecated? I see 5.0.0 for DryIoc.dll but the main DryIoc package is still at 4.8.8.

**Comment by dadhi** [comment]
@InKahootz No, source packages will be released soon.

**Comment by hymccord** [comment]
Thanks!

**Reply by dadhi** [reply]
Released Today :-)
